app/test_split_integer.py

```diff
@@ -2,20 +2,20 @@
 
 
 def test_sum_or_the_parts_should_be_equal_to_value():
-    pass
+    assert split_integer(8, 1) == [8]
```

Comment:
Uhm, this is not the check for this test. Did you read the name carefully?

Reply:
Actually, now that I have read the name carefully, I can see that there's a typo. I'll report that...

Reply:
totally agree